source/common/ipc/api.ts

```diff
@@ -492,8 +497,11 @@ export type signTransactionLedgerRendererRequest = LedgerSignTransactionRequest;
 export type signTransactionLedgerMainResponse = LedgerSignTransactionResponse;
 export const SIGN_TRANSACTION_TREZOR_CHANNEL =
   'SIGN_TRANSACTION_TREZOR_CHANNEL';
-export type signTransactionTrezorRendererRequest = TrezorSignTransactionRequest;
-export type signTransactionTrezorMainResponse = TrezorSignTransactionResponse;
+export type signTransactionTrezorRendererRequest = CommonParams &
+  CardanoSignTransaction;
+export type signTransactionTrezorMainResponse =
```

Comment:
```suggestion
export type SignTransactionTrezorRendererRequest = CommonParams &
  CardanoSignTransaction;
export type SignTransactionTrezorMainResponse =
```

Reply:
That's a valid comment @szymonmaslowski but we still would need to refactor the whole file. For now, I'd leave it as-is for consistency.